.github/workflows/build.yml

```diff
@@ -21,8 +21,18 @@ jobs:
       - name: add llvm
         run: |
             if [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
+
               sudo apt-get update
-              sudo apt-get install -y git gcc g++ #python3 python3-dev python3-pip
+              sudo apt-get install -y git gcc g++ gcc-12 g++-12 #python3 python3-dev python3-pip
```

Comment:
what if you remove the old gcc/g++

Reply:
let me try - I think it was an issue with it not being able to find headers